osquery/tables/system/linux/disk_encryption.cpp

```diff
@@ -23,6 +23,12 @@ extern "C" {
 
 #include "osquery/core/conversions.h"
 
+// FIXME: Add enum generation for tables and remove following code
+// Copy of values in disk_encryption.mm
+const std::string kEncryptionStatusEncrypted = "encrypted";
+const std::string kEncryptionStatusUndefined = "undefined";
+const std::string kEncryptionStatusNotEncrypted = "not encrypted";
```

Comment:
Looks like copy-paste  from `disk_encryption.mm`. Is here any way to use common code for it?

Reply:
It's copy-paste, this why there is FIXME there :)
It is very common use case so we need to solve it in general way, IMHO by providing enum generation as part of table generation